qutebrowser/browser/webkit/webkitelem.py

```diff
@@ -111,6 +111,25 @@ def tag_name(self) -> str:
         self._check_vanished()
         return self._elem.tagName().lower()
 
+    def attr(self, name: str) -> str:
+        """Get 'name' attribute if exists"""
+        return self._js_dict.get('attributes', {}).get(name, '')
+
+    def matches_hint_filter(self, filterstr: str, attrs=['text']) -> str:
```

Comment:
I think the implementation between webkit/webengine is the same here, so I think the concrete def can go in AbstractWebElement so we only need one (just make sure to use non-specific methods, which I don't think you are)